### ISSUE TITLE
Add an option to treat associates of "Verified Business" accounts like any other other Twitter Blue account

### ISSUE THREAD

**Issue by insin** [insin/control-panel-for-twitter#374]
When hiding boosted Blue replies, the ones I'm still seeing are mostly associates of "Verified Business" accounts, which we're still letting the "Actually Verified" ticks display on.

This isn't working, as "Verified Business" now just means someone who has more money than a Twitter Blue user - most of these seem to be associates of crypto accounts (which anyone can pay virtual Pogs to become an associate of) and other… dodgy organisations.

Add an option to also hide these accounts.

TL;DR: he was mad that blue checks were a status symbol (when their actual value was verification) and now they denote negative status on their bearer and are useless for verification. It's almost impressive.
